src/extension/ln/nostr/index.ts

```diff
@@ -41,6 +56,8 @@ export default class NostrProvider {
   // NOTE: new call `action`s must be specified also in the content script
   execute<T>(action: string, args?: Record<string, unknown>): Promise<T> {
     return new Promise((resolve, reject) => {
+      const id = Math.random().toString().slice(4);
```

Comment:
Would it be better to use an incremental ID? I think there would be no chance of collisions in that case

Reply:
incrementing requires state, can we rely on the state? 
I'd say the collision is unlikely.